core/src/edu/asu/c3simulator/screens/MainHub.java

```diff
@@ -0,0 +1,223 @@
+		navigation.setPosition(navigation.getWidth() / 2 - padding, 500);
+		System.out.println("Create new home screen");
+
+		stage.addActor(playerLabel);
+		// stage.addActor(advisor);
+		stage.addActor(company);
+		stage.addActor(navigation);
+		stage.addActor(Council());
+		// stage.addActor(playerInfo());
+
+	}
+
+	/**
+	 * Creates the business image in the middle of the screen. (Note: image is
+	 * not meant to be clickable)
+	 * 
+	 * @return Table
+	 */
+	private Actor Company()
+	{
+		Table company = new Table();
+		Image companyIcon;
+		float companyLeft;
+		float companyBottom;
+		FileHandle iconLocation = Gdx.files.internal("images/CompanyIcon.png");
+		Texture iconTexture = new Texture(iconLocation);
+		companyIcon = new Image(iconTexture);
+		company.add(companyIcon).size(800, 520);
+		// sets the height and width. Estimated that the image has 1/5 of white
+		// space on bottom, so actual image is upper 4/5.
+		companyLeft = DESIGN_WIDTH * 8 / 10 - companyIcon.getPrefWidth() / 2;
+		companyBottom = companyIcon.getHeight() * 4 / 5;
+		company.setPosition(companyLeft, companyBottom);
+
+		return company;
+	}
+
+	/**
+	 * Creates the council image in the bottom left of the screen. (Note: not
+	 * meant to be clickable)
+	 * 
+	 * @return Table
+	 */
+	private Actor Council()
+	{
+		Table council = new Table();
+		Image companyIcon;
+		float councilLeft;
+		float councilBottom;
+		FileHandle iconLocation = Gdx.files.internal("images/Council.png");
+		Texture iconTexture = new Texture(iconLocation);
+		companyIcon = new Image(iconTexture);
+		council.add(companyIcon).size(200);
+		// sets the image to the bottom left then shifts 50 pixels to the right
+		councilLeft = companyIcon.getPrefWidth() / 2 + 50;
+		councilBottom = companyIcon.getHeight();
+		council.setPosition(councilLeft, councilBottom);
+
+		return council;
+	}
+
+	/**
+	 * Creates an instance of player status display in the upper left corner of
+	 * the screen
+	 * 
+	 * @return Table
+	 */
+	private Actor playerInfo()
+	{
+		// TODO: Replace c3Simulation interface with actual backend, uncomment
+		// player
+		// status display when playerStatusDisplay reachest master_testing
+		/*
+		 * C3Simulation sim = new C3Simulation() { public String
+		 * getSimulationDate() { return ("Jan, Year: 1"); } };
+		 * PlayerStatusDisplay playerStatus = new PlayerStatusDisplay( new
+		 * PlayerImp("Plebian", 8000, 2000, sim));
+		 * 
+		 * playerStatus.setPosition(200, 600); return playerStatus;
+		 */
+		return null;
+	}
+
+	@Override
+	public void dispose()
+	{
+		stage.dispose();
+		this.game = null;
+	}
+
+	@Override
+	public void hide()
+	{
+		Gdx.input.setInputProcessor(null);
+	}
+
+	@Override
+	public void pause()
+	{
+		// TODO Auto-generated method stub
+		throw new UnsupportedOperationException(
+				"The method is not implemented yet.");
+	}
+
+	@Override
+	public void render(float delta)
+	{
+		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
```

Comment:
1: In the past there wasn't the option to use C3Simulation.clearScreen() - it is relatively new, and helps tie together our code.

2: Yes, line 166 works, however, using C3Simulation.clearScreen() has significant benefits to maintainability, including bug tracking, updating, and debugging. Also, it has added readability benefits thrown in for free.